src/main/java/redis/clients/jedis/BuilderFactory.java

```diff
@@ -421,6 +416,16 @@ public String toString() {
     }
   };
 
+  public static final Builder<KeyValue<Long, Long>> KEYED_LONG_LONG = new Builder<KeyValue<Long, Long>>() {
```

Comment:
`KEYED` prefix indicates first value of that complex object represents a Redis key. That's not the case here. So, rename it.

I.e. LONG_LONG_PAIR.

Reply:
Thanks, didn't get the convention at first, changed :)